src/com/zombiecastlerush/util/Parser.java

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
For some reason, my IDE gave me red paint under Parser until I used the auto-add option to give `package com.zombiecastlerush.util`. Suggest adding that.

Reply:
added package: package com.zombiecastlerush.util